lemarche/templates/siaes/detail.html

```diff
@@ -299,6 +299,26 @@ <h3 class="h3 mb-1 mt-1">Labels & certifications</h3>
                         {% endif %}
                     </div>
                     {% endif %}
+                    <!-- Sixth Row : Réalisations -->
+                    {% if siae.images.count %}
+                    <div class="row">
+                        <div class="col-12 img-left">
+                            <div class="il-l">
+                                <img src="{% static 'itou/images/ico-bicro-realisation.svg' %}" alt="" height="32" />
```

Comment:
avec le merge du thème ça va changer 
```suggestion
                                <img src="{% static_theme_images 'ico-bicro-realisation.svg' %}" alt="" height="32" />
```